dist/ui/czi-table.css

```diff
@@ -7,22 +7,19 @@
 }
 
 .ProseMirror table {
-  border-collapse: initial;
+  border-collapse: collapse;
```

Comment:
`border-collapse: collapse` was not used because ghost pixels appear after resizing table column.
After using the new column-resize plugin, the problem seems disappear.